src/encoded/static/components/cart/cart.js

```diff
@@ -2022,7 +2041,9 @@ const CartComponent = ({ context, savedCartObj, inProgress, fetch, session }) =>
                                         totalPageCount={totalPageCount.browser}
                                         updateCurrentPage={updateDisplayedPage}
                                     />
-                                    {Object.keys(selectedTerms).length > 0 ? <CartBrowser files={selectedVisualizableFiles} assembly={selectedTerms.assembly[0]} pageNumber={pageNumbers.browser} /> : null}
+                                    {Object.keys(selectedTerms).length > 0 && selectedTerms.assembly && selectedTerms.assembly.length > 0
```

Comment:
Why determine if selectedTerms have keys if all you need is selectedTerms.assembly; which you added a check for?

Reply:
Yeah, that’s true.